### DISCUSSION TITLE
Increment enum possible?

### DISCUSSION THREAD

**Discussion by thekalbb** [nguyenquyhy/Flight-Tracker-StreamDeck#270] [Ideas]
Would it be possible to impliment +1 / +10 / +100 / +1000 as an option on increase/decrease value - keys?

Using +1 on Airspeed is painfully slow when holding down the key.

is this allready a thing or can it be made?

Thanks for awsome software :)

**Comment by nguyenquyhy** [comment]
The plugin already has a kind of accelerated change when you hold the button for heading and altitude. However, I didn't do so for vertical speed and airspeed and instead just trigger the built-in INC/DEC event of the sim as there seems to be some existing accelerated logic there when I test around on the TBM. 

That might be wrong feeling due to the former +1/+10 incremental bug of the sim, so I'll retest this later.
